app/wind_plugins/sample/plugin.php

```diff
@@ -50,44 +50,37 @@ public function onUpgrade(string $prevVersion, string $curVersion, HookManager $
         }
 
         // 示例：根据版本差异执行不同的升级逻辑
-        // if (version_compare($prevVersion, '1.1.0', '<')) {
-        //     // 执行从1.1.0版本以下升级所需的特定操作
-        // }
+        //         if (version_compare($prevVersion, '1.1.0', '<')) {
+        //             // 执行从1.1.0版本以下升级所需的特定操作
+        //         }
     }
 
     public function activate(HookManager $hooks): void
     {
-        // 请求进入动作：依授权决定是否执行
+        // 请求进入动作：权限已在注册阶段检查，此处无需重复检查
```

Comment:
在sample插件中移除了手动权限检查，这是一个好的变更。现在权限检查在路由注册阶段完成，避免了在钩子回调中重复检查权限，提高了性能并简化了插件代码。

```suggestion
// 请求进入动作：权限已在注册阶段检查，此处无需重复检查
\app\service\PluginService::add_action('request_enter', function ($request) {
    // 演示：在请求对象上添加一个属性标记
    try {
        $request->sample_flag = true;
    } catch (\Throwable $e) {
        // 保持演示最小侵入，忽略异常
    }
}, 10, 1);